cachi2/core/package_managers/bundler/main.py

```diff
@@ -24,5 +24,12 @@ def fetch_bundler_source(request: Request) -> RequestOutput:
 
 def _resolve_bundler_package(package_dir: RootedPath, output_dir: RootedPath) -> list[Component]:
     """Process a request for a single bundler package."""
-    parse_lockfile(package_dir)
+    dependencies = parse_lockfile(package_dir)
+
+    # TODO: remove this hunk
+    deps_dir = output_dir.join_within_root("deps", "bundler")
+    for dep in dependencies:
+        if isinstance(dep, GitDependency):
```

Comment:
If you add a do-nothing `download_to` to _GemMetadata base class like in [PR-648](https://github.com/containerbuildsystem/cachi2/pull/648) (parser.py, L45) you would not need to check for type here.

Reply:
I will drop this...